anomalib/models/draem/loss.py

```diff
@@ -0,0 +1,204 @@
+"""Loss functions for DRAEM implementation."""
+
+# Original Code
+# Copyright (c) 2022 VitjanZ
+# https://github.com/VitjanZ/DRAEM.
+# SPDX-License-Identifier: MIT
+#
+# Modified
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+# pylint: disable=invalid-name
+
+from math import exp
+
+import numpy as np
+import torch
+import torch.nn.functional as F
+from torch import Tensor, nn
+
+
+class FocalLoss(nn.Module):
+    """Copy from: https://github.com/Hsuxu/Loss_ToolBox-PyTorch/blob/master/FocalLoss/FocalLoss.py.
```

Comment:
Please keep in mind that, for each 3rd party component we use, we need to add to `third-party-components.txt` in the main root